stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/AddressElement.kt

```diff
@@ -75,7 +75,7 @@ open class AddressElement constructor(
                     addressType = addressType,
                     isPlacesAvailable = isPlacesAvailable,
                 )
-                field.setRawValue(rawValuesMap)
+                field.setRawValue(currentValuesMap)
```

Comment:
This change and the one above are because the form was not preserving values after a refresh, like changing country, or in the case of ACH after connecting the bank, and it was because the element is basically being reloaded and filled with default values.